site/assets/scss/_navbar.scss

```
@@ -57,6 +57,12 @@
       color: $white;
     }
 
+    &:focus-visible {
```

```suggestion
    &:focus-visible {
      outline: $white solid 1px;
    }
```
It seems to do the trick quite well apart for the `.navbar-brand` on Safari but I don't know why 🤷